2-mandatory/4-stocks.js

```diff
@@ -64,7 +75,16 @@ function getPriceChanges(closingPricesForAllStocks) {
     The price should be shown with exactly 2 decimal places.
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
-    // TODO
+    const report = [];
+    for (let i = 0; i < closingPricesForAllStocks.length; i++) {
+        const stock = stocks[i].toUpperCase();
+        const prices = closingPricesForAllStocks[i];
+        const highestPrice = prices.reduce((maxPrice, price) => price > maxPrice ? price : maxPrice, -Infinity);
```

Comment:
I noticed that you often use reduce method. I only use it to calculate the sum of numbers. Perhaps on the next  session we would review cases where it is better to use reduce method.